moderatorFrontend/src/main/kotlin/views/users/UserTableRow.kt

```diff
@@ -113,9 +113,8 @@ class UserTableRow : RComponent<UserTableRowProps, UserTableRowState>() {
               if (window.confirm(Strings.user_delete_are_you_sure.get())) {
                 launch {
                   val response = NetworkManager.post<String>(
-                    "$apiBase/user/delete", params = json(
-                      "userId" to props.config.user.id
-                    )
+                    "$apiBase/user/delete",
+                    urlParams = mapOf("userId" to props.config.user.id)
```

Comment:
i am pretty sure that this should not be done via url params, but via post payload, ideally as json (as before)
(here and everywhere)

Reply:
Sorry, PR was created while still work in progress.

The code & naming was copied from the DAL PR and is very confusing... 😅 
`urlParams` actually means body form parameters in this case. So it's incorrectly labeled and also not what we need. We have changed the naming here and will backport it to DAL.

Reply:
No, the urlParams is correctly named, as it actually sets url query parameters.
![image](https://user-images.githubusercontent.com/11734608/129593666-8869bcc7-37e0-44c9-bf9f-6eb39c533d26.png)

Reply:
Yo will have to add an option to add form parameters though if needed, as it was not needed in DAL.

Reply:
Oh yes, thanks for the clarification regarding the urlParams, we didn't read it correctly. Form params shouldn't be needed right now.